src/controllers/request.controller.js

```diff
@@ -31,4 +31,32 @@ export default class RequestController {
       return errorResponse(res, badRequest, duplicateTripRequest);
     }
   }
+
+  /**
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} sends response to user
+   */
+  static placesAndVisitTimes = async (req, res) => {
+    const visitTimes = await getMostTraveledDestinations();
+    if (visitTimes[0].length !== 0) {
+      const destinations = visitTimes[0];
+      const destinationCount = destinations.map(dest => ({ Destination: dest.destination,
+        timeVisited: dest.count
+      }));
+      successResponse(res, ok, placesRetrieved, null, destinationCount);
+    } else {
+      errorResponse(res, notFound, noTripYet);
+    }
+  }
+
+  /**
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} sends response to user
+   */
+  static acceptRequest = async (req, res) => {
```

Comment:
What does this method do?

Reply:
@the22mastermind  This method is to help me approve a request to be able to pass tests, actually, approving request is not my task, but I need it because the default request status is `pending` so because of that I missed a way of testing if the route that I am working is working, this method will be deleted or modified with someone who will do the request approval, but now is needed for testing purpose!